src/wasm.h

```diff
@@ -1427,15 +1427,15 @@ class Function {
   }
 };
 
+enum class Kind {
```

Comment:
fwiw: the binary encoding doc calls it `external_kind`. The spec repo has them separate as `export_kind` and `import_kind`. I call it `ExternalKind` in wabt.